src/main/java/practice/StreamPractice.java

```diff
@@ -50,7 +75,15 @@ public List<Person> selectMenByAge(List<Person> peopleList, int fromAge, int toA
      */
     public List<Person> getWorkablePeople(int fromAge, int femaleToAge,
                                           int maleToAge, List<Person> peopleList) {
-        return Collections.emptyList();
+        Predicate<Person> personPredicate =
+                p -> p.getAge() >= fromAge
```

Comment:
Another way to implement
```suggestion
                p -> p.getSex() == Person.Sex.WOMAN
                        ? p.getAge() >= fromAge && p.getAge() <= femaleToAge
                        : p.getAge() >= fromAge && p.getAge() <= maleToAge
```